### DISCUSSION TITLE
Added Tenants do not appear

### DISCUSSION THREAD

**Discussion by neweratech-au-nn** [KelvinTegelaar/CIPP#822] [Q&A] [answered]
When adding new tenants in to our Partner Center, we don't get them to appear in CIPP unless we stop and restart  or we clear the cache.

Is this the way its is at present, or are we missing something in the process to add a new Tenant?

**Comment by KelvinTegelaar** [comment, marked as answer]
Tenants are cached for 24 hours since last retrieval, so it can take up to a day for it to cache out and add the tenant.  Clearing a cache just makes that a little bit faster :)
